files/__addonLocation__/tsconfig.json

```diff
@@ -48,6 +48,10 @@
       We want our tooling to know how to resolve our custom files so the appropriate plugins
       can do the proper transformations on those files.
     */
-    "allowImportingTsExtensions": true
+    "allowImportingTsExtensions": true,
+
+    "types": [
+      "./node_modules/ember-source/types/stable"
```

Comment:
Shouldn't we import this instead in `unpublished-development-types`, similar to what's suggested by the [official blog post on native types](https://blog.emberjs.com/stable-typescript-types-in-ember-5-1/)?

This might work, but I feel a bit bad about hardcoding the location on disk here. Better would be to just let node module resolution do its work, as in the case of the import. It is not guaranteed to be exacltly there in each and every scenario, like when this is part of a larger monorepo with hoisting, this could end up being in e.g. `../../node_modules/ember-source/types/stable` or something, right?

Reply:
> import

It doesn't work anymore 
https://github.com/emberjs/ember.js/issues/20789

> like when this is part of a larger monorepo with hoisting, this could end up being in e.g. ../../node_modules/ember-source/types/stable or something, right?

Maybe , but I belive this behavior is incorrect, and should be disabled, if possible

Reply:
> It doesn't work anymore
> https://github.com/emberjs/ember.js/issues/20789

That seems like a bug though!?

> I belive this behavior is incorrect

I don't think so. As long as the node module resolution algorithm is able to find it, should be ok?

If this were a .js, not .json, we could have done something like `require.resolve('ember-source/types/stable')`. But then, probably, importing would have continued to work in the first place?

Reply:
yeah, something is up, for sure.

I'm _hoping_ it's the addition of `package.json#exports` without `types` specified